packages/binding-coap/test/coap-server-test.ts

```diff
@@ -297,19 +297,32 @@ class CoapServerTest {
         let responseCounter = 0;
 
         const defaultContentFormat = "application/td+json";
-        const unsupportedContentFormat = "application/cbor";
-        const contentFormats = [defaultContentFormat, "application/json", unsupportedContentFormat];
+        const unsupportedContentFormat = "application/cbor"; // TODO: Update once CBOR is supported
```

Comment:
Wouldn't it be safer to just use a random string to check for unsupportedContent format? we could remove the TODO in this case.

Reply:
You're right, very good point :)

Reply:
Should be addressed now :)